rclcpp/test/rclcpp/test_time_source.cpp

```diff
@@ -516,3 +516,229 @@ TEST_F(TestTimeSource, no_pre_jump_callback) {
   EXPECT_EQ(1, cbo.last_postcallback_id_);
   EXPECT_EQ(1, cbo.post_callback_calls_);
 }
+
+// A TimeSource-inheriting class
+// that allows to access to TimeSource protected attributes
```

Comment:
```suggestion
// that allows access to TimeSource protected attributes
```

Reply:
Done in [last commit](https://github.com/ros2/rclcpp/pull/1556/commits/e16e18430eceaa5647a8b9e3e0447ff367e2973e)